src/library.rs

```diff
@@ -1085,18 +1085,19 @@ impl Library {
                 }
             }
             if let Some(tid) = env.library.find_type(0, &full_name) {
-                let gobject_id = env.library.find_type(0, "GObject.Object").unwrap();
+                let gobject_id = env.library.find_type(0, "GObject.Object");
```

Comment:
If you don't have `GObject` in your dependencies then there can't be any class hierarchy at all, so the code below should simply not run

Reply:
Ok gotcha